### PR TITLE
Tokens: removing px units from font size tokens

### PR DESCRIPTION
**Pull request by aarshap**
#### What changed?

We've removed px units from font size tokens. 

#### Why?

There are two reasons we've made this change:

1. Tokens serve multiple platforms, not all of which use px values.

2. Tokens Studio can't perform math on numerical values that contain a unit. We need unitless values for typography tokens that will be part of an upcoming release.

@rlingineni will be updating this PR with an update to our transformer that will apply platform-specific values on compile.

### CODE REVIEW

**Comment by rlingineni** [comment]
@diyorbek - this is a good example of a Style Dictionary transformer and how we're using it to add "px" in the outputs

File: packages/gestalt-design-tokens/build.js [hunk 1]
```diff
@@ -771,6 +787,7 @@ StyleDictionary.registerTransformGroup({
     'name/cti/snake',
     'name/conflictFixing',
     'color/hex8android',
```
Comment (AlbertCarreras) [inline packages/gestalt-design-tokens/build.js:789]: Be careful , in Android we have `sp`
Screenshot by Dropbox Capture

Comment (AlbertCarreras) [reply packages/gestalt-design-tokens/build.js:789]: iOS uses px.

**Review by AlbertCarreras — CHANGES_REQUESTED** [review]
see comment

**Review by AlbertCarreras — APPROVED** [review]

File: packages/gestalt-design-tokens/build.js (hunk 1 above)
Comment (rlingineni) [reply packages/gestalt-design-tokens/build.js:789]: Thanks! Yup this converts to px first.

All snapshots pass with the transformer

**Pull request merged**